upload/common/checksum_event.py

```diff
@@ -9,13 +9,25 @@
 
 class ChecksumEvent:
 
+    @classmethod
+    def load(cls, db_id):
+        db = UploadDB()
+        prop_vals_dict = db.get_pg_record("checksum", db_id)
+        return cls(
+            checksum_id=db_id,
+            job_id=prop_vals_dict['job_id'],
+            file_id=prop_vals_dict['file_id'],
+            status=prop_vals_dict['status']
+        )
+
     def __init__(self, **kwargs):
         self.job_id = kwargs.get("job_id")
         self.id = kwargs["checksum_id"]
-        self.file_id = kwargs["file_id"]
-        self.status = kwargs["status"]
+        self.file_id = kwargs.get("file_id")
```

Comment:
Why this change? Would you ever initialize a checksum event without passing in the status and file id?